demos/satija-2020/Snakefile

```diff
@@ -0,0 +1,92 @@
+include: "../common.smk"
+configfile: "config.yml"
+
+# URL constants
+CELL_ANNOTATIONS_URL = "https://vitessce-data.s3.amazonaws.com/source-data/satija/annotations_spleen_0721.csv"
+CELLS_URL = "https://vitessce-data.s3.amazonaws.com/source-data/satija/uf-processed.tar.xz"
+CL_OBO_URL = "https://vitessce-data.s3.amazonaws.com/source-data/satija/cl.obo"
```

Comment:
What is the provenance of these files?

Reply:
I just added comments in the file. The original files were obtained from emails in 2020 from Matt Ruffalo and people in the Satija lab.